<launch>  
  <node pkg="rbx1_vision" name="pos_calculator" type="pos_calculator.py" output="screen">
  
    <remap from="camera_info" to="/camera/rgb/camera_info" />
    <remap from="point_cloud" to="/camera/depth_registered/points" />
    <remap from="depth_image" to="/camera/depth_registered/image_raw" />

    <rosparam>
       rate: 10
       max_z: 4000
       min_z: 500
       scale_roi: 0.8
    </rosparam>
    
  </node>
</launch>
